### DISCUSSION TITLE
"For Enterprise" form

### DISCUSSION THREAD

**Discussion by vstarush** [payloadcms/payload#3045] [General]
Hello,

My name is Vitaly. I recently submitted an "For Enterprise" form and didn't get a confirmation email back. Just wanted to check if you received it, or there was some problem with the form submission?

If you didn't, could you please let me know what is the best way to reach you for the assistance?
Would really appreciate any feedback :)

Thank you!

**Comment by denolfe** [comment]
Thank you for following up. We respond to these by hand as they come in. We've just replied to you.
